test/CadlRanchProjects/azure/example/basic/tests/Generated/Samples/Samples_AzureExampleClient.cs

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using NUnit.Framework;
+using _Specs_.Azure.Example.Basic.Models;
+
+namespace _Specs_.Azure.Example.Basic.Samples
+{
+    public partial class Samples_AzureExampleClient
+    {
```

Comment:
This project is to test loading JSON example and generating sample code,  so the test and the sample should use the example value from JSON. Please config in tspconfig.yaml to set the `examples-dir`

Reply:
fixed